x/decentralized-vault/types/keys.go

```diff
@@ -0,0 +1,34 @@
+package types
+
+const (
+	// ModuleName defines the module name
+	ModuleName = "decentralizedvault"
+
+	// StoreKey defines the primary module store key
+	StoreKey = ModuleName
+
+	// RouterKey is the message route for slashing
+	RouterKey = ModuleName
+
+	// QuerierRoute defines the module's query routing key
+	QuerierRoute = ModuleName
+
+	// MemStoreKey defines the in-memory store key
+	MemStoreKey = "mem_decentralizedvault"
+)
+
+const (
+	WrappedClassName         = "ununifi wrapped"
+	WrappedClassId           = "ununifi-wrapped"
+	WrappedClassSymbol       = "ununifi-wrapped"
```

Comment:
symbol should be set to something I guess?

Reply:
We haven't finalized the specifications yet, so that's fine as it is.

Reply:
Okay